cosmo_tester/test_suites/cluster/conftest.py

```diff
@@ -276,6 +278,8 @@ def _ensure_installer_not_installed(vm):
             ]),
         )
     )
+    vm.run_command('if rpm -qi cloudify-manager-install; then '
```

Comment:
This is unnecessary- cloudify-manager-install is removed in the previous command.